ldes-server-domain/src/test/java/be/vlaanderen/informatievlaanderen/ldes/server/domain/tree/member/entities/MemberTest.java

```
@@ -72,4 +74,25 @@ private Model createModel(final String ldesMember, final Lang lang) {
 		return RDFParserBuilder.create().fromString(ldesMember).lang(lang).toModel();
 	}
 
+	@Nested
+	class GetMemberIdWithoutPrefix {
+		@Test
+		void shouldThrowException_whenIdHasNoPrefix() {
+			Member member = new Member(
+					"http://localhost:8080/member/1", "collectionName",
+					0L, null, null, null, List.of());
+
+			assertThrows(IllegalStateException.class, member::getMemberIdWithoutPrefix);
+		}
+
+		@Test
+		void shouldReturnIdWithoutPrefix_whenIdHasPrefix() {
+			Member member = new Member(
+					"parcels/http://localhost:8080/member/1", "collectionName",
```

This looks like an example that should throw an error no?